apps/src/templates/ProtectedStatefulDiv.jsx

```diff
@@ -32,7 +33,7 @@ var ProtectedStatefulDiv = React.createClass({
   },
 
   render: function () {
-    return <div {...this.props} ref="root"></div>;
+    return <div {..._.omit(this.props, ['contentFunction'])} ref="root"/>;
```

Comment:
Ah, sorry, there should be an explanation here.  I was getting [the Unknown Prop warning](https://facebook.github.io/react/warnings/unknown-prop.html) here because `contentFunction` (an optional function used by ProtectedStatefulDiv to generate its own innerHTML on mount) was getting forwarded to the `<div>` component here.  The warning was introduced in [React 15.2.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#1520-july-1-2016) which @pcardune _just_ [updated us to last week](https://github.com/code-dot-org/code-dot-org/commit/a3303432974a23543da7b63efb3789bf6593a40a).  Cleaning up React warnings is important because they 'guarantee' safe updates if your code is warning-free.